tests/src/test/resources/versions.conf

```diff
@@ -6,7 +6,8 @@ versions {
   plugins {
     pythonCommunity = { id = "PythonCore", version = "202.7319.50" }
     scala = { id = "org.intellij.scala", version = "2020.2.797", channel = "nightly" }
-    pants = { id = "com.intellij.plugins.pants", version = "1.15.1.23aef4e78f4a3073aed550d1a796328e28a34692", channel = "bleedingedge" }
+#    pants = { id = "com.intellij.plugins.pants", version = "1.15.1.23aef4e78f4a3073aed550d1a796328e28a34692", channel = "bleedingedge" }
+    pants = { id = "pants", repository.path = "https://github.com/tpasternak/intellij-pants-plugin", repository.ref = "synchronous-refresh" }
```

Comment:
I can't find the branch "synchronous-refresh", although the logs suggest that it had existed when the tests were run. Has it been deleted?